about.html

```diff
@@ -0,0 +1,150 @@
+              ></a>
+            </li>
+            <li class="nav-item"><a href="#" class="nav-link">English</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">My Page</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">Logout</a></li>
+          </ul>
+        </div>
+        <div class="header-bottom">
+          <div class="logo"><a href="./index.html">My Logo</a></div>
+          <div class="nav-right">
+            <ul class="main-nav">
+              <li class="nav-item">
+                <a href="./index.html" class="nav-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="#program" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+              <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campain">CG Campain</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <main class="about-wrapper">
+      <section class="about">
+        <div class="about-us">
+          <span class="about-quote"
+            >“Our highest deeds come from helping the lowest people.”</span
+          >
+          <h1 class="about-goal">
+            “Overcoming poverty is not a gesture of charity. It is an act of
+            justice. It is the protection of a fundamental human right, the
+            right to dignity and a decent life.”
+          </h1>
+        </div>
+        <div class="about-ideal">
+          The Genuine Charity brings together the community of experts,
+          academics, nad activists who comprise the Creative Commins affiliates
+          network in a different country every two years. This year, we hope to
+          expand our invitation list including organizations and individuals who
+          want to work with us on shared projects taht advance the cause of yhe
+          commons, free culture nd open kwnoledge. So if you're active and
+          engaged in the worlds of open content and kwnoledge - free software
+          advocates. Wikipedias, Open Kwnoledge, galeries, libraries, museums,
+          archives, governments and foundations, kawyers, and activists - we
+          hope you'll consider joing us this year to build a stronger, more
+          vibrant commons together.
+        </div>
+        <div class="contact">
+          Please contact us per Email for any further questions about Genuine
+          Charity and care at
+          <strong><u>christianbiringanine22@gmail.com</u></strong> >
```

Comment:
- [ ] **Required:** Kindly reposition this element.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/163601928-3cc76a78-f1eb-4530-950c-0742f7ef77ba.png) | ![image](https://user-images.githubusercontent.com/27709832/163601902-8fd1e62d-4f80-46fe-9ea4-876e3d565ab5.png)

Reply:
not resolved

Reply:
I have forgotten about that one.
Thanks again for remind me @Kingstalux